app/spike/internal/cmd/cipher/decrypt.go

```diff
@@ -0,0 +1,122 @@
+//    \\ SPIKE: Secure your secrets with SPIFFE. — https://spike.ist/
+//  \\\\ Copyright 2024-present SPIKE contributors.
+// \\\\\\ SPDX-License-Identifier: Apache-2.0
+
+package cipher
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	sdk "github.com/spiffe/spike-sdk-go/api"
+
+	"github.com/spiffe/spike/app/spike/internal/stdout"
+	"github.com/spiffe/spike/app/spike/internal/trust"
+)
+
+func newDecryptCommand(source *workloadapi.X509Source, SPIFFEID string) *cobra.Command {
+	var inFile string
+	var outFile string
+	var versionStr string
+	var nonceB64 string
+	var ciphertextB64 string
+	var algorithm string
```

Comment:
is there any reason defining these externally, instead of inside the `RunE` function?